forms/src/main/java/org/devgateway/toolkit/forms/application.properties

```diff
@@ -10,4 +10,10 @@
 # Development Gateway - initial API and implementation
 ###############################################################################
 spring.profiles.active=reports
-server.port = 8090
\ No newline at end of file
+server.port = 8090
+
+# Defines role hierarchy.
+# Hierarchy is specified as a string. Space separates rules and > symbol has the meaning of 'includes'.
+# Example: role1 > role2 > role3 role2 > role4
+# Here role1 includes role2 role3 and role4 (indirectly). And role2 includes role4.
+roleHierarchy=ROLE_ADMIN > ROLE_USER
```

Comment:
You are right. Initially when it was configured through code I tried to use constants for roles and these were defined in forms module. Now since it is defined through properties it does not depend on these constants. On the hindsight, maybe these constants could be moved to web module too?